discovery_engine_flutter/example/pubspec.lock

```diff
@@ -104,6 +104,27 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.4"
+  http:
```

Comment:
Why the lock is updated when no new deps are adde or modifed in this pr?

Reply:
I think it was supposed to be updated in with one of the previous PRs, namely #59, but not sure why `flutterp pub get` didn't pick it up before